TurkishFilter/sections/general_extensions.txt

```diff
@@ -32,6 +32,8 @@ technopat.net#%#(function() { var w_open = window.open, regex = /goo.gl/; window
 ! END: Clickunders
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/176425
+akakce.com#$#li.ignore { top: 0 !important; }
```

Comment:
CSS rule is not required here. I'd prefer this one

```suggestion
akakce.com##.billboard_content
```

or

```suggestion
akakce.com###CPL > li.ignore
```

Reply:
first one looks good to me - to block possible reuse of this `billboard_content`